src/Lib/PHPManagerConfiguration.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace PHPManager\PHPManager\Lib;
+
+final readonly class PHPManagerConfiguration
+{
+    function __construct(public string $phpManagerDirectory = '/.php-manager', public string $distDirectory = '/dist')
```

Comment:
I would put them on individual line for readability:

```suggestion
    function __construct(
        public string $phpManagerDirectory = '/.php-manager',
        public string $distDirectory = '/dist',
   )
```